johnktravers/olympics_api
# Import CSV data into database

### Pull Request Details
- Adds tests with validation and relationship tests for team, sport, olympian, event, and olympian_event models
- Adds team, sport, olympian, event, and olympian_event models and factories
- Creates `import:data` rake task to import CSV data into the database

### Relevant Issue(s)
- #2 Import CSV data into database

### Test Coverage
- 100%

### Manual testing
- Run `rake import:data` and check Postico to see that all tables are filled properly with CSV data

### Thoughts/Refactors
- It might be a good idea to break out the medal/height/weight error handling out of the rake task and into separate methods
